test/testvm.py

```diff
@@ -309,7 +309,7 @@ def upload(self, sources, dest):
         if self.verbose:
             subprocess.check_call(cmd)
         else:
-            with stdchannel_redirected(sys.stderr, os.devnull):
+            with stdchannel_redirected(sys.stdout, os.devnull), stdchannel_redirected(sys.stderr, os.devnull):
```

Comment:
This only affects upload - scp with the options listed above. No scripts are executed.